_po/ja/reference/modules/pgroonga-standby-maintainer.po

```diff
@@ -0,0 +1,151 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Language: ja\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+msgid ""
+"---\n"
+"title: pgroonga_standby_maintainer module\n"
+"upper_level: ../\n"
+"---"
+msgstr ""
+
+msgid "# `pgroonga_standby_maintainer` module"
+msgstr ""
+
+msgid "Since 2.4.2."
+msgstr "2.4.2で追加。"
+
+msgid "This is still an experimental feature."
+msgstr "まだ実験的な機能です。"
+
+msgid "## Summary"
+msgstr "## 概要"
+
+msgid ""
+"The `pgroonga_standby_maintainer` module provides executing [the `pgroonga_wal"
+"_apply()` function][pgroonga-wal-apply] and [the `pgroonga_vacuum()` function]"
+"[pgroonga-vacuum] automatically on stadnby database."
+msgstr ""
+"`pgroonga_standby_maintainer` モジュールは、 [`pgroonga_wal_apply()` 関数][pgroonga-wal"
+"-apply] と [`pgroonga_vacuum()` 関数][pgroonga-vacuum] をスタンバイのデータベース上で自動的に実行します。"
+
+msgid ""
+"Normally, if we use PGroonga with streaming replication, PGroonga's WAL doesn'"
+"t apply on a standby database.\n"
+"Therefore, for example, a first \"SELECT\" after we switch standby database to p"
+"rimary database may become slow.\n"
+"Because a time for applying not applicable PGroonga's WAL exist."
+msgstr ""
+"通常、PGroongaとストリーミングレプリケーションを一緒に使っている場合、PGroongaのWALはスタンバイのデータベースには適用されません。\n"
+"したがって、例えば、スタンバイのデータベースをプライマリーのデータベースに切り替えた後の最初の\"SELECT\"は遅い可能性があります。\n"
+"未適用のPGroongaのWALを適用する時間があるためです。"
+
+msgid ""
+"In addition, we need to remove internal unused Groonga tables, columns and rec"
+"ords with [the `pgroonga_vacuum()` function][pgroonga-vacuum] periodically on "
+"a standby database. Because `VACUUM` isn't run in a standby database."
```

Comment:
```suggestion
"a standby database. Because `VACUUM` isn't run on a standby database."
```

Execute on?
databasaeで実行することが onかinかどちらかわからないのですが 前出は　execute onだったので・・